### PR TITLE
Fixes CI

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Drawing tablets don't need frames.

## How This Contributes To The Skyrat Roleplay Experience
Hello, I'd like things to work again

### CODE REVIEW

**Review by GoldenAlpharex — APPROVED** [review]
Looks like that'd work, yeah.

File: code/modules/unit_tests/paintings.dm [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 /datum/unit_test/paintings/Run()
 	for(var/obj/item/canvas/canvas_prototype as anything in typesof(/obj/item/canvas))
+		//SKYRAT EDIT START
+		if(initial(canvas_prototype.type) == /obj/item/canvas/drawingtablet) //This doesn't need frames.
```
Comment (Iamgoofball) [inline code/modules/unit_tests/paintings.dm]: this is not how unit tests work

**Review by GoldenAlpharex — APPROVED** [review]
That's more like it.

File: code/modules/unit_tests/paintings.dm [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
 /datum/unit_test/paintings/Run()
 	for(var/obj/item/canvas/canvas_prototype as anything in typesof(/obj/item/canvas))
+		//SKYRAT EDIT START
+		if(canvas_prototype == /obj/item/canvas/drawingtablet) //This doesn't need frames.
```
Comment (Iamgoofball) [inline code/modules/unit_tests/paintings.dm:7]: this is still not how unit tests work, this is cheating at the test

**Pull request merged**